app/views/hyrax/una_books/_una_book.html.erb

```diff
@@ -0,0 +1,2 @@
+<%# This is a search result view %>
+<%= render 'catalog/document', document: una_book, document_counter: una_book_counter  %>
```

Comment:
You have double spaces at the end of these erb statements.

Reply:
I would assume this is how rails generates them then